library/src/scripts/components/mebox/pieces/DropDownMessageList.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import * as React from "react";
+import classNames from "classnames";
+import DropDownMessage, { IDropDownMessage } from "./DropDownMesssage";
+
+export interface IVanillaHeaderNavProps {
+    className?: string;
+    children: IDropDownMessage[];
+    emptyMessage?: string;
+}
+
+/**
+ * Implements Navigation component for header
+ */
+export default class DropDownMessageList extends React.Component<IVanillaHeaderNavProps> {
+    public render() {
+        const count = this.props.children.length;
+        const content = this.props.children.map((item, key) => {
+            return (
+                <React.Fragment key={`dropDownMessageList-${key}`}>
```

Comment:
With a single item these fragments should be unnecessary. You can just put the key on `DropDownMessage` directly.